tox.ini

```diff
@@ -93,12 +93,12 @@ commands =
     posix: python -c "print('Running on POSIX (no special dependencies)')"
 
     ; Run tests without wrapping them using coverage.
-    nocov: python -m twisted.trial --temp-directory={envtmpdir}/_trial_temp --reactor={env:TWISTED_REACTOR:default} --reporter={env:TRIAL_REPORTER:verbose} {env:TRIAL_ARGS:} {posargs:twisted}
+    nocov: python -m twisted.trial --temp-directory={envtmpdir}/_trial_temp --reporter={env:TRIAL_REPORTER:verbose} {env:TRIAL_ARGS:} {posargs:twisted}
```

Comment:
For greater flexibiltiy, if `--reactor` is needed, it is passed via `env:TRIAL_ARGS`

`env:TRIAL_REPORTER` is no longer used.

Reply:
Sorry. A typo. I editted the initial comment.

`env:TWISTED_REACTOR` is no longer used. You can set it via `TRIAL_ARGS`